index.js

```diff
@@ -54,6 +54,9 @@ function cljsLambdaBuild(serverless, opts) {
   const cmd = (`lein update-in :cljs-lambda assoc :functions '${fns}' ` +
                `-- cljs-lambda build :output ${serverless.service.__cljsArtifact} ` +
                `:quiet`);
+  // TODO if it is a lumo function use our lumo script for compiling
+  // const cmd = (`lumo -c serverless-cljs-plugin -m lumo.build` +
```

Comment:
This is the command that works, but you need `lumo` on `master` for it to work, I am patching `lumo`, see https://github.com/anmonteiro/lumo/pull/237